christfellowshipchurch/web-app-v2
# Update Share Messages

### About
We wanted to update the messaging for when sharing an article with the `Share` button. This PR updates the default messaging and adds separate messaging for sharing an event.

### Test Instructions
* Go to any **article** in mobile and press the share button and click on SMS, you should see the following: _"I thought you might be interested in this article from Christ Fellowship:"_
* Go to any **event** in mobile and press the share button and click on SMS, you should see the following: _"Join me for ${title} at Christ Fellowship!"_

### Screenshots
|Article|Event|
|---|---|
|![image](https://user-images.githubusercontent.com/46049974/130493911-f8e9e0cd-d22a-4270-922b-ebbaa82e95bc.png)|![image](https://user-images.githubusercontent.com/46049974/130494102-35acee44-b9cc-4bd5-9d22-105da403d792.png)|


### Closes Tickets
[CFDP-1670]
